build/plugins/analytics.html

```diff
@@ -9,3 +10,5 @@
   ga('send', 'pageview');
 </script>
 <!-- End Google Analytics -->
+
+<!-- copy and paste code from Google Analytics or similar service here -->
```

Comment:
Is this the intended code placement?  The Google Analytics code that was here is above the comment.  It probably doesn't matter enough to change.

Reply:
Yeah I saw that. I think `analytics.html` maybe should be updated so there is no blank line between the two comments and there is a final comment like `<!-- palce analytics above`

Reply:
Okay.  So it's an upstream change and nothing we need to do here.